.github/workflows/testing.yml

```diff
@@ -44,16 +44,21 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Enable test coverage
+        id: coverage
+        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.node == '18' }} # Node.js LTS
+        run: echo "enabled=true" >> "${GITHUB_OUTPUT}"
```

Comment:
See https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter